internal/pkg/manifest/lb_web_svc.go

```diff
@@ -187,8 +187,10 @@ func (s *LoadBalancedWebService) BuildRequired() (bool, error) {
 }
 
 // BuildArgs returns a docker.BuildArguments object given a ws root directory.
-func (s *LoadBalancedWebService) BuildArgs(wsRoot string) *DockerBuildArgs {
-	return s.ImageConfig.Image.BuildConfig(wsRoot)
+func (s *LoadBalancedWebService) BuildArgs(wsRoot string) map[string]*DockerBuildArgs {
```

Comment:
nit: I know this was there before 🙏 , but ideally the `manifest` package is completely unaware of concepts like "workspace". 

Instead, can we rename this parameter to and update the comment:
```suggestion
func (s *LoadBalancedWebService) BuildArgs(contextDir string) map[string]*DockerBuildArgs {
```
The clients can decide to pass the workspace root as the default context directory, but the manifest is much more flexible than that. By updating the parameter name, we are hinting at other clients that it doesn't need to be always the path to the workspace